app/assets/javascripts/discourse/app/routes/user-activity-pending.js

```diff
@@ -1,6 +1,13 @@
-import UserAction from "discourse/models/user-action";
-import UserActivityStreamRoute from "discourse/routes/user-activity-stream";
+import DiscourseRoute from "discourse/routes/discourse";
+import { observes } from "discourse-common/utils/decorators";
 
-export default UserActivityStreamRoute.extend({
-  userActionType: UserAction.TYPES.pending,
+export default DiscourseRoute.extend({
```

Comment:
This route doesn’t seem to be used anymore so I recycled it. :recycle:

Reply:
Looks like it [was added](https://github.com/discourse/discourse/commit/5bf8c31af40eb62c5051ae76a0657644abcdb348) in 2015 and we forgot this route when we added Reviewable?